.github/workflows/deployment-tests.yml

```
@@ -0,0 +1,23 @@
+  workflow_dispatch:
+
+jobs:
+  deployment-dev-tests:
+    uses: ./.github/workflows/deployment-test-template.yml
+    with:
+      compose_file: docker-compose.dev.yaml
+
+  deployment-prod-tests:
+    uses: ./.github/workflows/deployment-test-template.yml
+    with:
+      compose_file: docker-compose.yaml
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)